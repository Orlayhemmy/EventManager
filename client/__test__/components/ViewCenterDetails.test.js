import React from 'react';
import Details from '../../components/CenterDetails/Template/Content/ViewCenterDetails';

describe('Approve Event Component', () => {
  const onApprove = jest.fn();
  const showHiddenDiv = jest.fn();

  const centerState = {
    centerName: '',
    location: '',
    facilities: '',
    description: '',
    imageUrl: '',
    capacity: '',
    cost: ''
  };
  it('should render component without error', () => {
    const wrapper = renderer
      .create(<Details
          showHiddenDiv={showHiddenDiv}
          onApprove={onApprove}
          centerState={centerState}
        />)
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
