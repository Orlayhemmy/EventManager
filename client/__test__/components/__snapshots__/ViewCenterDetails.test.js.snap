// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Approve Event Component should render component without error 1`] = `
<div
  id="centerDetails"
>
  <div
    className="imageUpload"
  >
    <img
      className="img-fluid dropzone"
      src=""
    />
  </div>
  <div
    className="media-body text-center mb-4 mt-4"
  >
    <strong
      className="logo text-primary mb-2"
    >
      
    </strong>
    <h3
      className="mt-2"
    >
      location
    </h3>
    <p>
      
    </p>
    <h3
      className="mt-2"
    >
      capacity
    </h3>
    <p>
      
    </p>
    <h3
      className="mt-2"
    >
      facilities
    </h3>
    <p>
      
    </p>
    <h3
      className="mt-2"
    >
      price
    </h3>
    <p>
      
    </p>
    <h3
      className="mt-2"
    >
      description
    </h3>
    <p>
      
    </p>
  </div>
  ...
   
  <i
    className="fa fa-pencil-alt main-color"
    data-toggle-id="editCenterDetails"
    onClick={[MockFunction]}
  >
    edit
  </i>
</div>
`;
